kubebuilder.mk

```diff
@@ -68,6 +72,17 @@ vet:
 generate:
 	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
 
+.PHONY: protos
+protos:
```

Comment:
I think we need this to be part of some kind of more generic step that can be run prior to committing code. Maybe something similar to the `make update` process in k/k. I think that would need to include `generate`, `protos`, `fmt`, `vet`, and `manifests`. Not completely tied to specifics here, just that this PR doesn't add any additional steps in the service-apis development process. Ideally only 1 command would be required to generate all code required (currently that's `make all` but mirroring `make update` from k/k may make more sense).

Reply:
I agree, I added and `update` target which adds (I think) everything needed